<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SusanForm</class>
 <widget class="QMainWindow" name="SusanForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SUSAN-2D Corner Detector</string>
  </property>
  <property name="windowIcon">
   <iconset resource="truecornerdetector.qrc">
    <normaloff>:/icons/icon.ico</normaloff>:/icons/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>281</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListWidget" name="fileListWidget">
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideRight</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>30</number>
         </property>
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeButton">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="horizontalSpacing">
          <number>4</number>
         </property>
         <property name="verticalSpacing">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="QSlider" name="sliderT">
           <property name="maximumSize">
            <size>
             <width>185</width>
             <height>22</height>
            </size>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QSlider" name="sliderW">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>22</height>
            </size>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1850</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>T</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>t</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBoxT">
           <property name="maximumSize">
            <size>
             <width>53</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinBoxFinalT">
           <property name="maximumSize">
            <size>
             <width>53</width>
             <height>20</height>
            </size>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1850</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="gboxWindow">
         <property name="minimumSize">
          <size>
           <width>256</width>
           <height>81</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>256</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Non-maxima suppression window</string>
         </property>
         <widget class="QRadioButton" name="radN1">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>82</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>3x3</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QRadioButton" name="radN3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>82</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>5x5</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="radN2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>82</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>4x4</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="checkBoxSmoothing">
           <property name="text">
            <string>Use smoothing</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Corners detected: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelNCorners">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="maximumSize">
            <size>
             <width>108</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Save corners</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>0</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="textVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="viewTabs">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>680</width>
        <height>500</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>fileListWidget</tabstop>
 </tabstops>
 <resources>
  <include location="truecornerdetector.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>sliderT</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxT</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxT</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderT</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderW</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxFinalT</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxFinalT</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderW</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
